saleor/graphql/menu/types.py

```diff
@@ -0,0 +1,23 @@
+from graphene import relay
+
+from ...menu import models
+from ..core.types import CountableDjangoObjectType
+
+
+class Menu(CountableDjangoObjectType):
+    class Meta:
+        description = """Represents a single menu - an object that is used
+        to help navigate through the store."""
+        interfaces = [relay.Node]
+        filter_fields = ['id', 'name']
```

Comment:
Do we need the filtering by `id` here, since we have an endpoint to fetch a particular menu? Also, since we've introduced `query` in some of the other queries, I'd prefer to stay consistent with them and drop filtering by `name` in favor of `query`. We could stick to that convention in all queries.

Reply:
I am not sure about this one. `query` was intended to search in _many_ fields like name, address, country etc, for a quick filter. Here we have one field, so I would go for `name__Icontains` - it is technically the same, but we remain consistent at least.

Reply:
In my opinion, `query` sounds like a good name for any kind of text-based searching/matching/filtering - it's easy to understand when you look at it and definitely more user-friendly than `name__Icontains`. To be honest I'm not a fan of this `django-filters` integration, mostly because of the ugly looking argument names it generates. Maybe we could customize it. Also, I've read that a good practice is to keep the arguments list as short as possible.